chaincode/input.go

```diff
@@ -90,6 +90,20 @@ type inputTraintuple struct {
 	Tag            string   `validate:"omitempty,lte=64" json:"tag"`
 }
 
+// inputCompositeTraintuple is the representation of input args to register a Traintuple
+type inputCompositeTraintuple struct {
+	AlgoKey                string           `validate:"required,len=64,hexadecimal" json:"algoKey"`
+	ObjectiveKey           string           `validate:"required,len=64,hexadecimal" json:"objectiveKey"`
+	InHeadModelKey         string           `validate:"required_with=InTrunkModelKey,omitempty,len=64,hexadecimal" json:"inHeadModelKey"`
```

Comment:
seems that you inverted model keys (trunk/head)

Reply:
@inalgnu if you're talking about the `required_with` field, it's intentional! It's to make sure you can't pass a `head` model without `trunk` model, and vice versa.

Reply:
my bad I thought it was the json field name.